### PR TITLE
Cu 862j1hec4 cloud admin component s18288pejotedupl

### PR DESCRIPTION
**Pull request by s18288**

### CODE REVIEW

**Comment by s20146** [comment]
Task linked: [CU-862j1hec4 CloudAdminComponent](https://app.clickup.com/t/862j1hec4)

File: React/src/Pages/Orders.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Orders = () => {
+    return <div>Orders</div>
+}
+
+export default Orders;
```
Comment (kurylek) [inline React/src/Pages/Orders.tsx]: On master we have Orders list working with api

File: React/src/Pages/WarehouseList.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const WarehouseList = () => {
+    return <div>WarehouseList</div>
+}
+
+export default WarehouseList;
```
Comment (kurylek) [inline React/src/Pages/WarehouseList.tsx]: Same here

File: React/src/components/CloudAdmin/CloudAdmin.tsx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                    </TableContainer>
+                </Grid>
+            </Grid>
+        </Box>
+    );
+
+    return content;
+}
+
+export default CloudAdmin;
```
Comment (kurylek) [inline React/src/components/CloudAdmin/CloudAdmin.tsx]: It should be done like this:
https://github.com/s20146/PRO/tree/master/React/src/pages/orders

File: React/src/types/roleEnum.ts [hunk 4]
```diff
@@ -3,10 +3,10 @@ export enum Role {
...
-    WAREHOUSE_MAN = 'WAREHOUSE_MAN',
-    WAREHOUSE_MANAGER = 'WAREHOUSE_MANAGER',
-    FITTER = 'FITTER',
-    FOREMAN = 'FOREMAN'
+    MERCHANT = 'MERCHANT',
+    EXPERT = 'EXPERT',
+    WAREHOUSEMAN = 'WAREHOUSEMAN',
+    WAREHOUSE_MANAGER = 'WAREHOUSE MANAGER',
+    ASSEMBLER = 'ASSEMBLER',
+    BRIGADE_LEADER = 'BRIGADE_LEADER'
```
Comment (kurylek) [inline React/src/types/roleEnum.ts]: Old/Bad role names

File: React/src/utils/pageList.tsx [hunk 5]
```diff
@@ -44,205 +42,157 @@ const Root = () => {
...
-                        name: 'Lista zleceń',
-                        path: '/orders'
-                    },
-                    {
-                        inNav: true,
-                        name: 'Dodaj zlecenie',
-                        path: '/orders/new'
-                    }
+
+export const pageList: Array<PageProps> =
```
Comment (kurylek) [inline React/src/utils/pageList.tsx]: It contains old/bad role names

**Review by kurylek — CHANGES_REQUESTED** [review]
It should be done like this: https://github.com/s20146/PRO/tree/master/React/src/pages/orders
Check quick guide pined id Teams chat

File: React/src/Pages/Orders.tsx (hunk 1 above)
Comment (s21461) [reply React/src/Pages/Orders.tsx]: Oj bedzik bedzik

File: React/src/utils/pageList.tsx [hunk 6]
```diff
@@ -18,7 +18,7 @@ import Schedules from '../pages/Schedules'
 import LoginPage from '../pages/LoginPage'
 import { Role } from '../types/roleEnum'
 import Error from '../components/error/Error'
-import Orders from '../pages/orders'
+import Orders from '../pages/Orders'
```
Comment (kurylek) [inline React/src/utils/pageList.tsx]:
```suggestion
import Orders from '../pages/orders'
```

**Review by kurylek — APPROVED** [review]

**Pull request merged**